Automattic/wp-calypso
# Removes the difm-design step

#### Changes proposed in this Pull Request

* Removes the `difm-design` step. This step has been now replaced `difm-design-setup-site` step in the `do-it-for-me` signup flow.

#### Testing instructions



* Start the DIFM Lite signup flow by navigating to `/start/do-it-for-me`.
* Confirm you are able to complete the flow.
* Confirm that there are no JS errors.



Related to 0-as-1201604657043036/f
